app/scripts/modules/core/src/deploymentStrategy/DeploymentStrategySelector.tsx

```diff
@@ -83,8 +83,15 @@ export class DeploymentStrategySelector extends React.Component<
     this.selectStrategy(this.props.command.strategy, true);
   }
 
+  public onChange = (key: string, value: any) => {
+    this.props.onFieldChange(key, value);
+
+    // This hack is needed to force the component to re-render when the angular deployment-strategy-selector updates the underlying command
```

Comment:
I believe this statement is true, but I don't see how `onChange` gets called when the underlying command is updated?

Reply:
`onChange` is called which then changes the underlying command via `onFieldChange`. The data changes but the component doesn't re-render. (`forceUpdate` will be good here).

Reply:
My question was asking HOW `onChange` is called by angularjs code when it updates the command object.  
NBD, it's probably not important as long as it works, but I just don't see it.